crates/rattler_conda_types/src/repo_data/mod.rs

```diff
@@ -19,14 +19,16 @@ use serde_with::{serde_as, skip_serializing_none, OneOrMany};
 use thiserror::Error;
 use url::Url;
 
-use crate::utils::serde::sort_map_alphabetically;
 use crate::utils::url::add_trailing_slash;
 use crate::{
     build_spec::BuildNumber,
     package::{IndexJson, RunExportsJson},
     utils::serde::DeserializeFromStrUnchecked,
     Channel, NoArchType, PackageName, PackageUrl, Platform, RepoDataRecord, VersionWithSource,
 };
+use crate::{
+    utils::serde::sort_map_alphabetically, MatchSpec, Matches, ParseMatchSpecError, ParseStrictness,
```

Comment:
From a quick glance I cant find where this util is used.

Reply:
wdym? i'm using these things (except `sort_map_alphabetically` which is used above in `RepoData`) in `PackageRecord::validate`